floor/floor/controller/midi/manager.py

```diff
@@ -33,7 +33,7 @@ class MidiManager(object):
     """Binds a pymidi server to the dance floor controller."""
     def __init__(self, port, controller, default_midi_mapping=None):
         self.controller = controller
-        self.midi_server = pymidi_server.Server(port=port)
+        self.midi_server = pymidi_server.Server([('0.0.0.0', port)])
```

Comment:
Now using the new version of pymidi, need to pass in a host tuple rather than just port.